contracts/ExitHandler.sol

```diff
@@ -309,16 +309,16 @@ contract ExitHandler is DepositHandler {
         tokens[currentExit.color].addr.transferFrom(address(this), currentExit.owner, currentExit.amount);
       }
       // Pay exit stake
-      currentExit.owner.transfer(currentExit.stake);
+      address(uint160(currentExit.owner)).transfer(currentExit.stake);
```

Comment:
what is going on here? isn't `currentExit.owner` already an `address` ?

Reply:
there is a difference between `address` and `address payable` now, only second one can be transfered to. in can only be cast from `uint160`. 🤷‍♂️